charts/consul/Chart.yaml

```diff
@@ -13,9 +13,11 @@ annotations:
   artifacthub.io/prerelease: true
   artifacthub.io/images: |
     - name: consul
-      image: hashicorp/consul:1.14.0-beta1
+      image: hashicorp/consul:1.14.0
     - name: consul-k8s-control-plane
       image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.0.0-dev
```

Comment:
Remove dev?

Reply:
This will be removed as part of `prepare-release` step 2 in https://github.com/hashicorp/engineering-docs/blob/main/consul/consul-k8s/release-process.md#creating-a-new-long-lived-release-branch but I can update it here

Reply:
Ah got it, thank you!